src/NServiceBus.Core/Licensing/LogErrorOnInvalidLicenseBehavior.cs

```diff
@@ -0,0 +1,19 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using Logging;
+    using Pipeline;
+
+    class LogErrorOnInvalidLicenseBehavior : Behavior<PhysicalMessageProcessingContext>
```

Comment:
Yeah, definitely. If I read correctly it was not really tested before your change so this PR would definitely make it better.